### ISSUE TITLE
Remember last-run monitor (like lazer)

### ISSUE THREAD

**Issue by solstice23** [ppy/osu-stable-issues#1009]
I have two monitors and only play osu! on the second monitor which has a higher framerate. Every time I start osu! from the start menu, it appears on my primary monitor, so I have to press alt+enter and move it to another monitor. Can we remember the last monitor like lazer?

Additional, the system hotkey win+shift+right won't take effect, because once you click away from the osu! window, it will be reset to the primary monitor.

**Comment by peppy** [comment]
osu already does this. are you not running fullscreen?

**Comment by solstice23** [comment]
No, I play in fullscreen mode. But I can always reproduce this. Forgive me for using this curt way to demonstrate because there are black screens in OBS during the few seconds after osu! starts. 

https://user-images.githubusercontent.com/23134847/177543175-28c3c745-61b8-4652-9c9d-e562aa6022af.mp4

(I exited osu! at the second monitor, and restarted it, it appeared in my primary monitor.)
